[2023-04-20 22:42:02,502] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2023-04-19T00:00:00+00:00 [queued]>
[2023-04-20 22:42:02,505] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2023-04-19T00:00:00+00:00 [queued]>
[2023-04-20 22:42:02,505] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-20 22:42:02,505] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-04-20 22:42:02,505] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-20 22:42:02,510] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): also_run_this> on 2023-04-19 00:00:00+00:00
[2023-04-20 22:42:02,512] {standard_task_runner.py:52} INFO - Started process 56800 to run task
[2023-04-20 22:42:02,514] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2023-04-19T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', '/Users/brenda/miniconda3/lib/python3.10/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/tf/f0rvty4s1yd90__k0gtqx_5h0000gn/T/tmpqhs1w6c2', '--error-file', '/var/folders/tf/f0rvty4s1yd90__k0gtqx_5h0000gn/T/tmpp9bgwjtz']
[2023-04-20 22:42:02,515] {standard_task_runner.py:80} INFO - Job 6: Subtask also_run_this
[2023-04-20 22:42:02,538] {task_command.py:369} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2023-04-19T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2023-04-20 22:42:02,561] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2023-04-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-19T00:00:00+00:00
[2023-04-20 22:42:02,562] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tf/f0rvty4s1yd90__k0gtqx_5h0000gn/T
[2023-04-20 22:42:02,562] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2023-04-19T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2023-04-19 00:00:00+00:00: scheduled__2023-04-19T00:00:00+00:00, externally triggered: False>"']
[2023-04-20 22:42:02,567] {subprocess.py:85} INFO - Output:
[2023-04-20 22:42:02,569] {subprocess.py:92} INFO - run_id=scheduled__2023-04-19T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2023-04-19 00:00:00+00:00: scheduled__2023-04-19T00:00:00+00:00, externally triggered: False>
[2023-04-20 22:42:02,570] {subprocess.py:96} INFO - Command exited with return code 0
[2023-04-20 22:42:02,580] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20230419T000000, start_date=20230420T204202, end_date=20230420T204202
[2023-04-20 22:42:02,621] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-20 22:42:02,633] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
